.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
```

Comment:
We should run tools using the lowest supported version to avoid compatibility issues - e.g.: our new CS release adds property type declaration when running on PHP 7.4

Reply:
Do new PHP versions enable new sniffs that aren't needed to be checked in older versions?

Reply:
Don't enable new sniffs but modify sniffs behaviour, like the property type declaration I mentioned.